wa/workloads/antutu/__init__.py

```diff
@@ -90,12 +114,24 @@ def __init__(self, device, **kwargs):
         super(Antutu, self).__init__(device, **kwargs)
         self.gui.timeout = 1200
 
+    def initialize(self, context):
+        super(Antutu, self).initialize(context)
+        #Install the supporting benchmark
+        supporting_apk = context.get_resource(ApkFile(self, package='com.antutu.benchmark.full'))
+        self.target.install(supporting_apk)
+        #Ensure the orientation is set to portrait
+        self.target.set_rotation(0)
```

Comment:
I'm assuming it should not, but would setting this change the behaviour for any previous versions of the workload?

Reply:
No, this is just to ensure the results page is in the correct orientation for the grabbing of results.